drivers/hmis/app/graphql/types/hmis_schema/unit.rb

```diff
@@ -64,7 +67,32 @@ def occupancy_status
 
     def deletable
       return false if occupants.any? # cannot delete unit with occupants
-      return false if load_ar_association(object, :active_referral).present? # cannot delete unit with active referral
+      return false if load_ar_association(object, :referrals).any? # cannot delete unit with any referrals (past or present)
```

Comment:
We can discuss this further (and maybe split out a separate ticket?) if we want users to be able to delete units that have past referrals.

Right now it fails because of `dependent: :restrict_with_exception` [here](https://github.com/greenriver/hmis-warehouse/blob/1bc5961608adfce4f2dbe75a3e1344e216fcde99/drivers/hmis/app/models/hmis/ce/opportunity.rb#L17)